<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDProjektClass</class>
 <widget class="QMainWindow" name="CDProjektClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CDProjekt</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="UsernameEdit">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>90</y>
      <width>300</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="UsernameTEXT">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>85</y>
      <width>120</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>账号</string>
    </property>
   </widget>
   <widget class="QLabel" name="PasswordTEXT">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>195</y>
      <width>120</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>密码</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PasswordEdit">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>200</y>
      <width>300</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="LogInButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>320</y>
      <width>200</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
    <property name="icon">
     <iconset theme="但是"/>
    </property>
   </widget>
   <widget class="QLabel" name="Copyright">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>440</y>
      <width>331</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Copyright @2021 BoomHan Studio V1.0.0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="CDProjekt.qrc"/>
 </resources>
 <connections/>
</ui>
